Demo/index.js

```diff
@@ -345,7 +345,7 @@ basic.set\`Hello {wrld}\`
     <div>Create a random string using letters and/or number and/or symbols.
     You may use this method for example to create password strings, or random element id's.</div>
     <p><b>Syntax</b>:<br><code class="codeBlock">$S.randomString({
-  length: Number( default 12),
```

Comment:
using [instance](`somestring`) would take the length of the first parameter (a string)